calyx-py/test/builder_example.py

```diff
@@ -0,0 +1,36 @@
+from calyx.builder import Builder
+
+
+def build():
+    prog = Builder()
+    main = prog.component("main")
+
+    lhs = main.reg("lhs", 32)
+    rhs = main.reg("rhs", 32)
+    sum = main.reg("sum", 32)
+    add = main.add("add", 32)
+
+    with main.group("update_operands") as update_operands:
```

Comment:
TIL this is possible. I always assumed that "update_operands" would be scoped to the block below?

Reply:
Honestly it was a surprise to me as well.